src/controller/java/src/matter/controller/CompletionListenerAdapter.kt

```diff
@@ -52,12 +52,8 @@ class CompletionListenerAdapter(val listener: MatterController.CompletionListene
 
   override fun onICDRegistrationInfoRequired() = listener.onICDRegistrationInfoRequired()
 
-  override fun onICDRegistrationComplete(
-    errorCode: Int,
-    icdNodeId: Long,
-    icdCounter: Long,
-    icdDeviceInfo: ICDDeviceInfo
-  ) = listener.onICDRegistrationComplete(errorCode, icdNodeId, icdCounter, icdDeviceInfo)
+  override fun onICDRegistrationComplete(errorCode: Int, icdDeviceInfo: ICDDeviceInfo) =
```

Comment:
ICDDeviceInfo is defined in Java, that prevent us from decoupling kotlin lib and java lib, we need to remove this decoupling

Reply:
As you know, I am also working on a separate implementation of the kotlin library and java library JNI. If I modify it now, I think that most Interface Classes (ICDDeviceInfo etc.) in Java should be implemented identically in Kotlin, and the only difference is the package name. Therefore, the Java package name is now used in kotlin-controller, but I think it would be better to change it while implementing the kotlin class.